src/components/BlockchainList/CommentWithReplies.tsx

```diff
@@ -71,12 +71,11 @@ function makeComment(post?: PostStructOutput, cast?: Cast) {
 
   if (cast) {
     return {
-      id: cast.merkleRoot,
-      replyToId: cast.merkleRoot,
-      replier: `@${cast.body.username}`,
-      replierAddress: cast.body.address,
-      content: cast.body.data.text,
-      timestamp: (cast.body.publishedAt / 1000) ^ 0,
+      id: cast.hash,
+      replyToId: cast.hash,
+      replier: `@${cast.author.username}`,
+      content: cast?.text,
+      timestamp: (cast.timestamp / 1000) ^ 0,
```

Comment:
We should extract those calculations into `helper`

Reply:
Here we need to refactor the application, it takes more time, let's add a task